src/tools/wpa/Tables/EventStats.cs

```diff
@@ -34,6 +34,16 @@ public sealed class EventStats
 
         public static void BuildTable(ITableBuilder tableBuilder, IDataExtensionRetrieval tableData)
         {
+            if (tableBuilder == null)
+            {
+                throw new ArgumentNullException(nameof(tableBuilder));
+            }
+
+            if (tableData == null)
+            {
+                throw new ArgumentNullException(nameof(tableData));
+            }
```

Comment:
Why are you adding these?

Reply:
Because this is a public function, and FxCop requires public functions to explicitly null check their variables. It throws an error otherwise. The reason is because the stacktrace for actually dereferencing null is harder to read then explicitly null checking.

Reply:
I prefer to global ignore that error. I don't want to add all these null checks.